com.writerduet.writersolo.yaml

```diff
@@ -0,0 +1,42 @@
+app-id: com.writerduet.writersolo
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+command: writersolo
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --device=dri
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+  - --filesystem=xdg-download
+rename-desktop-file: writersolo.desktop
+rename-icon: writersolo
+modules:
+
+  - name: writersolo
+    buildsystem: simple
+    build-commands:
```

Comment:
This block should be in `apply_extra` script. 

```yaml
    sources:
      - type: script
        dest-filename: apply_extra
        commands:
           - command
```

In build commands, you want this https://github.com/flathub/com.amazon.Workspaces/blob/7006428fe14a357d0146bd66968aa2c198b8eee9/com.amazon.Workspaces.yml#L132-L140